### ISSUE TITLE
Searching/Identifying Based on Crafted Mods

### ISSUE THREAD

**Issue by JeordyR** [Kyusung4698/PoE-Overlay#59]
Testing price check on Paradoxica and PoE-Overlay is unable to find a match when selecting any of the crafted mods. Confirmed with a few random rares that trying to filter on their crafted mods also does not return any results.

Version Tested: 0.4.3

Steps to Reproduce:
* Ctrl+D on an item with crafted mods
* Click one of the crafted mods in the pop-up results window

Example items:
```
Rarity: Unique
Paradoxica
Vaal Rapier
--------
One Handed Sword
Quality: +28% (augmented)
Physical Damage: 51-203 (augmented)
Critical Strike Chance: 6.50%
Attacks per Second: 1.50 (augmented)
Weapon Range: 14
--------
Requirements:
Level: 66
Dex: 212
--------
Sockets: G-R-R 
--------
Item Level: 83
--------
+25% to Global Critical Strike Multiplier (implicit)
--------
Attacks with this Weapon deal Double Damage
10% chance to Trigger Level 1 Blood Rage when you Kill an Enemy (crafted)
105% increased Physical Damage (crafted)
15% increased Attack Speed (crafted)
20% chance to Blind Enemies on hit (crafted)
--------
What has no siblings but is always a twin?
```

---

```
Rarity: Rare
Apocalypse Paw
Assassin's Mitts
--------
Quality: +20% (augmented)
Evasion Rating: 166 (augmented)
Energy Shield: 32 (augmented)
--------
Requirements:
Level: 65
Dex: 45
Int: 45
--------
Sockets: G-G-B-G 
--------
Item Level: 83
--------
Trigger Commandment of Ire when Hit
--------
Socketed Gems are Supported by Level 20 Faster Attacks
+15% to Chaos Damage over Time Multiplier
13% increased Attack Speed
40% increased Evasion and Energy Shield
+72 to maximum Life
+26% to Fire Resistance
23% increased Damage during any Flask Effect (crafted)
--------
Elder Item
```

**Comment by Kyusung4698** [comment]
Will be continued here #68

**Comment by Kyusung4698** [comment]
Fixed with 0.5.0-alpha.0.
